src/helpers/a11yAPI.js

```diff
@@ -6,11 +6,11 @@ import makeQuery from './makeQuery';
 type GetReturn = ReactTestInstance;
 type GetAllReturn = Array<ReactTestInstance>;
 type QueryReturn = ReactTestInstance | null;
-type QueryAllReturn = Array<ReactTestInstance> | [];
```

Comment:
Removed the `[]` fallback, as Flow treats it as a tuple and thinks that the array has no elements. Which may be true, but would require users to optional-chain a lot